src/Adapter/Configuration/LogsConfiguration.php

```diff
@@ -62,6 +76,17 @@ public function getConfiguration()
     public function updateConfiguration(array $configuration)
     {
         if ($this->validateConfiguration($configuration)) {
+            $checkEmails = explode(',', $configuration['logs_email_receivers']);
+            $errors = [];
+            foreach ($checkEmails as $email) {
+                if (!$this->validate->isEmail($email)) {
+                    $errors[] = $this->translator->trans('Invalid email address %email%.', ['%email%' => $email], 'Admin.Notifications.Error');
```

Comment:
Hi, 

Is it possible that there is more than one invalid email? If yes, we could use transChoice(

Invalid email: %invalid_emails%.|Invalid emails: %invalid_emails%.

Just like we did it here: https://github.com/PrestaShop/PrestaShop/pull/24622

We would also avoid creating 2 different strings to say the same thing.

Reply:
@Julievrz I made the change